pkg/signing/signing.go

```diff
@@ -235,88 +443,82 @@ func SifAddSignature(sinfo *image.Sifinfo, signature []byte) error {
  * 6) record the KeyID used to sign into signature data object descriptor
  */
 func Sign(cpath string) error {
```

Comment:
From `golint`:

```
pkg/signing/signing.go:434:1: comment on exported function Sign should be of the form "Sign ..."
```

Reply:
Sure, I assume there's a lot more do to than that to be honest.